client/src/interface/manageBooking.ts

```diff
@@ -0,0 +1,30 @@
+export interface props {
```

Comment:
Name isn't great, ManageBookingProps. Remember interfaces should follow CamelCase